File: lib/src/main/java/com/sd/lib/dialogview/core/handler/IDialogViewHandler.java
package com.sd.lib.dialogview.core.handler;

import com.sd.lib.dialogview.DialogView;

public interface IDialogViewHandler<T extends DialogView>
{
    int getContentView(T dialogView);

    void onContentViewChanged(T dialogView);

    void onAttachedToWindow(T dialogView);

    void onDetachedFromWindow(T dialogView);
}

File: lib/src/main/java/com/sd/lib/dialogview/core/handler/IDialogViewHandler.kt
package com.sd.lib.dialogview.core.handler

import com.sd.lib.dialogview.DialogView

interface IDialogViewHandler<T : DialogView> {
    /**
     * 返回内容布局id
     */
    fun getContentViewResId(dialogView: T): Int

    /**
     * 内容view变化
     */
    fun onContentViewChanged(dialogView: T)

    fun onAttachedToWindow(dialogView: T)

    fun onDetachedFromWindow(dialogView: T)
}